tests/sqlparser_snowflake.rs

```diff
@@ -2846,3 +2846,32 @@ fn test_parse_show_columns_sql() {
     snowflake().verified_stmt("SHOW COLUMNS IN TABLE abc");
     snowflake().verified_stmt("SHOW COLUMNS LIKE '%xyz%' IN TABLE abc");
 }
+
+#[test]
+fn test_sf_double_dot_notation() {
+    snowflake().verified_stmt("SELECT * FROM db_name..table_name");
+    snowflake().verified_stmt("SELECT * FROM x, y..z JOIN a..b AS b ON x.id = b.id");
+}
+
+#[test]
+fn test_sf_double_dot_notation_wrong_position() {}
```

Comment:
ah did we plan to put in a test scenario for this behavior?

Reply:
Ya, I left it by mistake. Will remove.